h/views/api/group_members.py

```diff
@@ -71,6 +71,18 @@ def list_members(context: GroupContext, request):
     return {"meta": {"page": {"total": total}}, "data": membership_dicts}
 
 
+@api_config(
+    versions=["v1", "v2"],
+    route_name="api.group_member",
+    request_method="GET",
+    link_name="group.member.read",
+    description="Fetch a group membership",
+    permission=Permission.Group.READ,
+)
+def get_member(context: GroupMembershipContext, request):
+    return GroupMembershipJSONPresenter(request, context.membership).asdict()
```

Comment:
:man_shrugging: 

If some of the old nonsense in h was factored out so that Pyramid could be used properly this could be even simpler:

```python
@view_defaults(...)
class MembershipAPIViews:
    ...

    @view_config(request_method="GET", permission=Permission.Group.READ, renderer=MembershipJSONRenderer)
    def get(context, request):
        return context.membership

    ...
```